src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -33,22 +64,33 @@
 
 <script setup>
 import { ref, computed } from 'vue';
+import { MaInput } from "@mobileaction/action-kit";
+import "@mobileaction/action-kit/dist/style.css"
```

Comment:
external CSS files should be defined in the `style` tag. Also the action-kit style file should be defined on a project basis, not on a component basis.